#!/usr/bin/env python
PACKAGE = "yolo_tf"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("grid_width", int_t, 0, "Number of columns in output layer", 12, 1, 30)
gen.add("grid_height", int_t, 0, "Number of rows in output layer", 9, 1, 30)
gen.add("threshold", double_t, 0, "Threshold of objectness", 0.1, 0., 1.)

exit(gen.generate(PACKAGE, "yolo", "YoloDetector"))
